packages/support/package.json

```diff
@@ -56,7 +56,6 @@
     "@types/npmlog": "4.1.4",
     "@types/pluralize": "0.0.29",
     "@types/pngjs": "6.0.1",
-    "@types/rimraf": "3.0.2",
```

Comment:
rimraf is still present in dependencies below

Reply:
Thanks for pointing it out, I assumed it was present as devDependencies towards the end of the file.

Reply:
Made the necessary changes